server/.golangci.yml

```diff
@@ -38,8 +40,17 @@ issues:
       text: "SetupEnterprise"
 
     - linters:
-      - golint
-      text: "should have|should be|should replace|stutters|underscore|annoying|error strings should not be capitalized"
+      - revive
+      text: "var-naming|error-naming|exported|increment-decrement|error-strings|if-return|unused-parameter|blank-imports|context-as-argument"
+      # We need to fix the unused parameter issues and remove the exception.
+
+    - linters:
+      - revive
+      path: "public/*" # TODO: fix this
+
+    - linters:
+      - revive
+      path: "enterprise" # TODO: fix this
```

Comment:
Do we have follow up tickets ready?

Reply:
It will be the same ticket. I'll finish it this Friday OSF.

Reply:
Created https://mattermost.atlassian.net/browse/MM-55409 :wink:

Reply:
Thanks for taking care of this Ben!